validation/upgrade/kubernetes_wins_test.go

```diff
@@ -68,7 +69,7 @@ func (u *UpgradeWindowsKubernetesTestSuite) TestUpgradeWindowsKubernetes() {
 			}
 
 			testConfig := clusters.ConvertConfigToClusterConfig(&cluster.ProvisioningInput)
-			upgradeDownstreamCluster(&u.Suite, tt.name, tt.client, cluster.Name, testConfig, cluster, tt.nodeSelector)
```

Comment:
why are we removing the nodeSelector from here? last I remembered was that removing this would cause the workloads to be scheduled on linux nodes as well for windows clusters.

Reply:
It wasn't being used in the function, and thus golangci-lint was complaining about it

Reply:
ah ok, thanks!